QRCoder/Builders/Renderers/RendererBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using QRCoder.Builders.Payloads;
+
+namespace QRCoder.Builders.Renderers
+{
+    public abstract class RendererBase : IRenderer, IConfigurableQuietZones
+    {
+        protected QRCodeData QrCodeData { private set; get; }
+
+        protected bool QuietZone { get; set; } = true;
+        bool IConfigurableQuietZones.QuietZone { get => QuietZone; set => QuietZone = value; }
+        IPayload IRenderer.Payload { set => QrCodeData = value.ToMatrix(); }
```

Comment:
Another option would be to store the `IPayload` object itself.  I opted to call `ToMatrix` immediately, so if there were any exceptions while converting the payload to a module matrix, it would be caught at this point, enhancing the ability to diagnose faulty fluent syntax.